<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stage_edit</class>
 <widget class="QWidget" name="stage_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>831</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1078</width>
        <height>811</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="5">
          <widget class="QComboBox" name="stages_tab_bgmusic_combo"/>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="stages_tab_stage_combo"/>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Stage Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_48">
           <property name="text">
            <string>Stage:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>Stage Music:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <widget class="QLineEdit" name="stages_tab_stage_name_lineedit">
           <property name="maxLength">
            <number>30</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="QComboBox" name="cutscenePre_comboBox"/>
         </item>
         <item row="0" column="3">
          <widget class="QComboBox" name="cutscenePos_comboBox"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Scene Before:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Scene After:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="3" column="0">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Boss Dialogs (Boss Room)</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_13">
              <item row="5" column="1">
               <widget class="QLineEdit" name="boss_dialog_text2_line3">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maxLength">
                 <number>30</number>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="boss_dialog_text1_line1">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maxLength">
                 <number>30</number>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="boss_dialog_text1_line2">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maxLength">
                 <number>30</number>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_24">
                <property name="text">
                 <string>Phrase 2:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="boss_dialog_text1_line3">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maxLength">
                 <number>30</number>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="boss_dialog_text2_line2">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maxLength">
                 <number>30</number>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLineEdit" name="boss_dialog_text2_line1">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maxLength">
                 <number>30</number>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>Phrase 1:</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>Player Answers (Boss Room)</string>
             </property>
             <layout class="QGridLayout" name="gridLayout_14">
              <item row="3" column="1">
               <widget class="QLineEdit" name="boss_dialog_answer2_line1">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maxLength">
                 <number>30</number>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Player Answer 1:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Player Answer 2:</string>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="5" column="1">
               <widget class="QLineEdit" name="boss_dialog_answer2_line3">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maxLength">
                 <number>30</number>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="boss_dialog_answer2_line2">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maxLength">
                 <number>30</number>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="boss_dialog_answer1_line2">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maxLength">
                 <number>30</number>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="boss_dialog_answer1_line3">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maxLength">
                 <number>30</number>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="boss_dialog_answer1_line1">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maxLength">
                 <number>30</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Language:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="language_comboBox"/>
             </item>
            </layout>
           </item>
           <item row="2" column="1">
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QLabel" name="label_55">
               <property name="text">
                <string>Player Answer #1:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="dialogs_answer1_text3">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="maxLength">
                <number>30</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="dialogs_answer1_text2">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="maxLength">
                <number>30</number>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_57">
               <property name="text">
                <string>Player Answer #2:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="dialogs_answer2_text3">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="maxLength">
                <number>30</number>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="dialogs_answer2_text2">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="maxLength">
                <number>30</number>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="dialogs_answer2_text1">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="maxLength">
                <number>30</number>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="dialogs_answer1_text1">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="maxLength">
                <number>30</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_54">
               <property name="text">
                <string>Dialog Player:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="dialogs_answer1_player"/>
             </item>
            </layout>
           </item>
           <item row="2" column="0">
            <layout class="QGridLayout" name="gridLayout_10">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_3">
               <property name="topMargin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="label_51">
                 <property name="text">
                  <string>Dialog #1:</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="2">
                <widget class="QLineEdit" name="dialogs_line2_text3">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="maxLength">
                  <number>30</number>
                 </property>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>Dialog #2:</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="2">
                <widget class="QLineEdit" name="dialogs_line1_text3">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="maxLength">
                  <number>30</number>
                 </property>
                </widget>
               </item>
               <item row="4" column="2">
                <widget class="QLineEdit" name="dialogs_line2_text2">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="maxLength">
                  <number>30</number>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLineEdit" name="dialogs_line1_text1">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="maxLength">
                  <number>30</number>
                 </property>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QLineEdit" name="dialogs_line2_text1">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="maxLength">
                  <number>30</number>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLineEdit" name="dialogs_line1_text2">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="maxLength">
                  <number>30</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_7">
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item row="0" column="1">
              <widget class="QLineEdit" name="stages_tab_bossname_lineedit">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="maxLength">
                <number>30</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Stage boss name:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Dialog Person Face:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QComboBox" name="dialogs_line1_face_combo">
               <property name="iconSize">
                <size>
                 <width>32</width>
                 <height>32</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>scrollArea</tabstop>
  <tabstop>stages_tab_stage_combo</tabstop>
  <tabstop>language_comboBox</tabstop>
  <tabstop>stages_tab_bossname_lineedit</tabstop>
  <tabstop>dialogs_line1_face_combo</tabstop>
  <tabstop>dialogs_answer1_player</tabstop>
  <tabstop>dialogs_line1_text1</tabstop>
  <tabstop>dialogs_line1_text2</tabstop>
  <tabstop>dialogs_line1_text3</tabstop>
  <tabstop>dialogs_line2_text1</tabstop>
  <tabstop>dialogs_line2_text2</tabstop>
  <tabstop>dialogs_line2_text3</tabstop>
  <tabstop>boss_dialog_text1_line1</tabstop>
  <tabstop>boss_dialog_text1_line2</tabstop>
  <tabstop>boss_dialog_text1_line3</tabstop>
  <tabstop>boss_dialog_text2_line1</tabstop>
  <tabstop>boss_dialog_text2_line2</tabstop>
  <tabstop>boss_dialog_text2_line3</tabstop>
  <tabstop>dialogs_answer1_text1</tabstop>
  <tabstop>dialogs_answer1_text2</tabstop>
  <tabstop>dialogs_answer1_text3</tabstop>
  <tabstop>dialogs_answer2_text1</tabstop>
  <tabstop>dialogs_answer2_text2</tabstop>
  <tabstop>dialogs_answer2_text3</tabstop>
  <tabstop>boss_dialog_answer1_line1</tabstop>
  <tabstop>boss_dialog_answer1_line2</tabstop>
  <tabstop>boss_dialog_answer1_line3</tabstop>
  <tabstop>boss_dialog_answer2_line1</tabstop>
  <tabstop>boss_dialog_answer2_line2</tabstop>
  <tabstop>boss_dialog_answer2_line3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
